Integration/ApiTestCase.php

```diff
@@ -0,0 +1,46 @@
+<?php
```

Comment:
This is specific to RocketCDN, at least in its current form

Reply:
Currently, yes. However, as we potentially move modules into SaaS, this particular test case could be expanded as a common `ApiTestCase`.

Reply:
Shouldn't we update this to make it a common test case right now, in this case?

Reply:
I see what you mean. It literally is only for RocketCDN right now. As such, it should be moved into that module and not here. You are 100% right. Sorry about that.

As it's extending off of `WPMedia\PHPUnit\Integration\RESTfulTestCase`. I think your original instinct here is right: build a specific use case when you need it.

I'll remove this one from this package. Good job!

Reply:
Removed in this commit https://github.com/wp-media/phpunit-wp-rocket/pull/1/commits/7d1f3ae8aabc5a5280c8d8ad1be65d1ab7d51693